modules/src/movie.py

```diff
@@ -23,23 +23,27 @@ def process(input, entities):
                 'include_adult': False
             })
             data = r.json()
-
             assert (len(data['results']) > 0)
             tmdb_id = str(data['results'][0]['id'])
-
             # Make another request to the API using the movie's TMDb ID to get the movie's IMDb ID
             r = requests.get('https://api.themoviedb.org/3/movie/' + tmdb_id, params={
                 'api_key': TMDB_API_KEY
             })
             data = r.json()
-
+            videoRequests = requests.get('https://api.themoviedb.org/3/movie/' + tmdb_id + '/videos', params={
```

Comment:
Why did we move to making a separate call? Does `append_to_response` not solve our use-case?